.gitignore

```diff
@@ -16,6 +16,9 @@
 !/pootle/translations/tutorial/
 !/pootle/translations/templates/
 
+# /pootle/pootle_fs/
+/pootle/pootle_fs/*/
```

Comment:
I don't this is a problem. Some similar would happen with `/pootle/translations/` IMHO.